lib/fathead/npm_cli/cover/redirects.txt

```diff
@@ -0,0 +1,74 @@
+npm access public, access
+npm access restricted, access
```

Comment:
@rahulxxarora however, it looks like we don't have redirects for the cli commands that are joined with a dash. E.g.
- npm-run-script
- npm-scripts
- npm-test
- npm-start
- npm-restart

We should have redirects for those, and in those cases `npm` needs to be part of the redirect title 👍